Android/Jackson.Core/nuget/Jackson.Core.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Jackson.Core</id>
+    <title>Jackson Core for Xamarin Android</title>
+    <version>2.6.1</version>
+    <authors>Xamarin Inc.</authors>
+    <owners>Xamarin Inc.</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>This project contains core low-level incremental ("streaming") parser and generator abstractions used by Jackson Data Processor.</description>
+    <copyright>Copyright 2016 Xamarin Inc</copyright>
+    <projectUrl>https://github.com/xamarin/XamarinComponents/tree/master/Android/Jackson.Core</projectUrl>
+    <licenseUrl>https://github.com/xamarin/XamarinComponents/blob/master/Android/Jackson.Core/License.md</licenseUrl>
+  </metadata>
+  <files>  
+    <file src="output/Jackson.Core.dll" target="lib/MonoAndroid23" />
```

Comment:
TBH I am guessing.  This is not an Android library from what [I can tell.](https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-core/2.8.5/jackson-core-2.8.5.pom)  Also I do not get any more (or less) warnings from classparse when the api level is changed.